packages/language-server/test/plugins/typescript/features/DocumentSymbolsProvider.test.ts

```diff
@@ -104,7 +104,7 @@ describe('TypeScript Plugin#DocumentSymbolsProvider', () => {
 				containerName: 'Template',
 				kind: SymbolKind.Constant,
 				location: {
-					range: Range.create(4, 9, 5, 25),
+					range: Range.create(4, process.platform === 'win32' ? 10 : 9, 5, 25), // NOTE: For some reason, the location of this is different on Windows, might be a bug?
```

Comment:
I don't know why that is, why it worked before and why only on that particular variable. Lots of mystery that I only vaguely feel like solving right now. It's only off by a single character, so functionality is still the same